docs/documentation.rst

```diff
@@ -318,67 +318,40 @@ opening the file ``qc/index.html``. Use the "Search"
 feature of the QC report to quickly jump to segmentations or labeling
 results.
 
-Segmentation
-^^^^^^^^^^^^
+Segmentation and vertebral labeling
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-If you spot segmentation issues, manually fix them using the procedure described
+If you spot segmentation or labeling issues, manually fix them using the procedure described
 below. Also see the video tutorial after the procedure.
 
-- Go to the ``results/data`` folder
-- Create the file ``fix_seg.sh`` and copy/past the code below:
+- Go to the ``results/data`` folder.
+- In the QC report, enter the string "deepseg" to only display segmentation results or the string "vertebrae" to only
+  display vertebral labeling.
+- Review all spinal cord and gray matter segmentations and vertebral labeling. Use the keyboard shortcuts up/down arrow
+  to switch between subjects and the left arrow to toggle overlay.
+- If you spot *major* issues with the segmentation (e.g. noticeable leaking or under-segmentation that extends over
+  several slices) or wrong labeling, add the image name into the yaml file as in an example below:
 
-.. code-block:: bash
+.. Hint::
+    You can create yaml file easily using your text editor (vim, nano, atom, ...).
+
+::
+
+    FILES_SEG:
+    - sub-amu01_T1w_RPI_r.nii.gz
+    - sub-amu01_T2w_RPI_r.nii.gz
+    - sub-cardiff02_dwi_moco_dwi_mean.nii.gz
+    FILES_GMSEG:
+    - sub-amu01_T2star_rms.nii.gz
+    FILES_LABEL:
```

Comment:
The files for vertebral labeling should be the image that the labeling is done on.

Reply:
Okay, but in help of `manual_correction.py` script are also listed sub ID, not files that labeling is done on:

https://github.com/spine-generic/spine-generic/blob/4a2ec8b194037107a2fc63373853efce54b67885/spinegeneric/cli/manual_correction.py#L55
https://github.com/spine-generic/spine-generic/blob/4a2ec8b194037107a2fc63373853efce54b67885/spinegeneric/cli/manual_correction.py#L56

Reply:
> Okay, but in help of `manual_correction.py` script are also listed sub ID, not files that labeling is done on:
> 
> https://github.com/spine-generic/spine-generic/blob/4a2ec8b194037107a2fc63373853efce54b67885/spinegeneric/cli/manual_correction.py#L55
> 
> 
> https://github.com/spine-generic/spine-generic/blob/4a2ec8b194037107a2fc63373853efce54b67885/spinegeneric/cli/manual_correction.py#L56

Can you pls fix it— thanks!

Reply:
Fixed in https://github.com/spine-generic/spine-generic/pull/127. You can merge it into master.